manifests/base/kustomization-ci.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ./crds
+  - ./rbac/cluster-role-binding.yaml
+  - ./rbac/role.yaml
+  - ./rbac/service-account.yaml
+  - ./webhook
+  - service.yaml
+  - deployment.yaml
+  - ./monitoring/network-policy.yaml
+  - ./monitoring/metrics-service.yaml
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)